activation/atxdb.go

```diff
@@ -341,6 +397,22 @@ func (db *ActivationDb) GetEpochAtxIds(epochId types.EpochId) ([]types.AtxId, er
 	return atxs, nil
 }
 
+// getAtxUnlocked gets the atx from db, this function is not thread safe and should be called under db lock
+// this function returns a pointer and
```

Comment:
```suggestion
// this function returns a pointer
```
(not sure if you wanted to add something here)